### ISSUE TITLE
configure zlog via C API

### ISSUE THREAD

**Issue by honkiko** [HardySimpson/zlog#82]
It's sometimes useful to configure zlog via C API.
One option is zlog_init_from_string(const char *) besides initialization from file. More fine granularity of API is also desirable.
If you think it's a good direction, I would be glad to contribute.

**Comment by HardySimpson** [comment]
Yes it is a good direction, want it very much. Thanks for contribute!

**Comment by mfide** [comment]
is it possible to init zlog without external file?

**Comment by yegorich** [comment]
Fixed upstream. Can be closed. See zlog_init_from_string / zlog_reload_from_string functions.
